db/fixtures/miq_policy_sets.yml

```diff
@@ -207,3 +207,225 @@
           action_type: default
           options: {}
       Condition: []
+- MiqPolicySet:
+    name: VM SmartState Analysis profile
+    description: VM SmartState Analysis profile
+    set_type: MiqPolicySet
+    guid: 2d7c4b41-474f-43dd-888d-8f3ddd5151a5
+    read_only: true
+    set_data:
+    mode: control
+    owner_type:
+    owner_id:
+    userid:
+    group_id:
+    MiqPolicy:
+    - name: eea4ca5c-68a3-11e7-ab6f-beeffeed0072
```

Comment:
Why are the names guids?

Reply:
That's how they're named when I exported the policies and profiles from the UI

Reply:
Can you give them proper names like the other ones in this file?